apps/wesh/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.7"
+
+services:
+  web:
+    image: ibz0/wesh:v0.2.1
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - 7681:7681
+    volumes:
+      - ${LND_DATA_DIR}:/lnd:ro
+    environment:
+        LND_HOST: "${LND_IP}"
+        LND_ADDRESS: "//${LND_IP}:10009"
```

Comment:
Are both LNDHOST and LND_ADDRESS needed? Seems like LND_HOST is redudant.

Reply:
Ahh these are for seperate apps, right?

Maybe this would be better handled by a wrapper in the Docker container. So wesh just accepts the minimum args it needs, and then the app specific args are built inside the container.